pipeline/launcher.py

```diff
@@ -54,6 +54,15 @@ def get_server_command(server_type, num_gpus):
         # somehow on slurm nemo needs multiple tasks, but locally only 1
         if CLUSTER_CONFIG["cluster"] == "local":
             num_tasks = 1
+
+    elif server_type == 'vllm':
+        server_start_cmd = (
+            f"(NUM_GPUS={num_gpus} bash /code/nemo_skills/inference/server/serve_vllm.sh /model/ {model_name} "
```

Comment:
Can you please remind me why model_name needs to be explicit here? I remember you were telling me before, but cannot recall what's the reason now

Reply:
It doesn't need to be if we don't put the logic for prompt handling anymore. It's just good for the user logs to show which model is loaded at this point so that they don't mistake which model is loaded and waste compute